src/hotspot/os/linux/os_linux.cpp

```diff
@@ -4477,6 +4478,21 @@ void os::Linux::numa_init() {
   }
 }
 
+// Check numa dependent syscalls
+bool os::Linux::numa_syscall_check() {
```

Comment:
This can just be a static function in the file rather then being in the os::Linux "namespace"

Reply:
> This can just be a static function in the file rather then being in the os::Linux "namespace"

Updated.
Thanks.